### ISSUE TITLE
FYI: updates to the new (nested) team structure

### ISSUE THREAD

**Issue by kytrinyx** [exercism/meta#13]
TL;DR:

1. Do not be alarmed when I remove you from the 'track-maintainers' team, you'll still have access to it if you're a maintainer of a specific track.
2. The track-maintainers team is not the right team to mention for cross-cutting concerns. To help decide how to manage these, please see https://github.com/exercism/discussions/issues/162

Ok, the longer version.

@exercism/track-maintainers Recently we changed the team structure to make all individual track teams nested under the track-maintainers team:

```
track-maintainers/
├── bash
├── # 61 more teams
└── vimscript
```

**I'm going to remove all the explicit memberships from the track-maintainers team**, since everyone who maintains a specific track is now implicitly a member of that team.

This has one slightly unfortunate side effect, which is that you can't opt out of team mentions to that team simply by leaving the team. I think we need to remove the need for regular team mentions to that team, and I've opened an issue to discuss how to best do that: https://github.com/exercism/discussions/issues/162

**Comment by kytrinyx** [comment]
Closing, as I hope you have all seen this by now :heart:
